spec/acceptance/install_spec.rb

```diff
@@ -5,14 +5,14 @@
     it 'installs influxdb' do
       pp = <<-MANIFEST
         include influxdb
-        MANIFEST
+      MANIFEST
 
       idempotent_apply(pp)
     end
 
     # Influxdb should be listening on port 8086 by default
-    describe port('8086') do
-      it { is_expected.to be_listening }
```

Comment:
What is the reasoning behind replacing the serverspec test with an exec?

Reply:
SLES 15 failures because no `netstat`, we had to make the same change in operational dashboards https://github.com/puppetlabs/influxdb/pull/29#issuecomment-1215483822